### PR TITLE
Support "Import Reflection" proposal

### PR DESCRIPTION
**Pull request by fisker**
## Description

Ref #13750

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: changelog_unreleased/javascript/13771.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#### Add support for "Import Reflection" proposal (#13771 by @fisker)
+
+The Stage 2 proposal ["Import Reflection"](https://github.com/tc39/proposal-import-reflection) is now supported via Babel 7.20.0. Also keep in mind our [policy on non-standardized syntax](https://github.com/tc39/proposal-explicit-resource-management/) before using this proposed syntax feature with Prettier.
```
Comment (liuxingbaoyu) [inline changelog_unreleased/javascript/13771.md:3]: Why link `policy on non-standardized syntax` to `proposal-explicit-resource-management`?😕

Comment (fisker) [reply changelog_unreleased/javascript/13771.md]: Copy-pasted from old changelog.

**Review by sosukesuzuki — APPROVED** [review]

**Comment by kachkaev** [comment]
@fisker WDYT of including this into 2.8?

**Comment by sosukesuzuki** [comment]
I don't think we need to add this to 2.8 because this is a still experimental ECMAScript feature and almost users don't need this.

**Comment by fisker** [comment]
Agree

**Pull request merged**